README.md

```diff
@@ -175,19 +175,17 @@ The output will be in `target/html/master.html`.
 
 See [benchmarking.md](benchmarking.md) for information on running basic performance tests for this proxy.
 
-### GitHub action for performance
+### Jenkins pipeline for performance
 When a PR is created and the performance tests are needed, the following comment shall be added into the PR:
 
 ```
-/perf
+@strimzi-ci run perf
 ```
 
-It will launch the `Performance` build, compare the results with the previous execution and store the results in the gh-pages repo under the `.benchmark` directory.
+It will launch the [kroxylicious-performance-tests-pr](https://master-jenkins-csb-amq-streams.apps.ocp-c1.prod.psi.redhat.com/job/kroxylicious/job/kroxylicious-performance-tests-pr/) build, that will insert a comment with a summary into the PR comparing the results with the previous execution.
 
-In case the results are worse than expected (150% threshold) and alert will be shown into the commit and an email will be sent to the kroxylicious developers
```

Comment:
Let me make sure I have this right.  So the job doesn't judge whether performance is bad and fail the job if that is true?

Reply:
That's right. This caused some troubles in the past where the 99th were worse than expected but the rest fine. Peaks can cause the 99th going higher but this may not be wrong if the rest are fine. We would need to make more accurate the pass/fail criteria if we want to add it to the pipeline

Reply:
We should add a sentence to the readme so the developer knows its on him/her to check the numbers.  Might be worth adding a tick item to the PR template too so we are mindful of the need to check and react to the numbers.

Reply:
Added a warning for that. Thanks!

Reply:
> Peaks can cause the 99th going higher but this may not be wrong if the rest are fine. We would need to make more accurate the pass/fail criteria if we want to add it to the pipeline

I think we do care about our p99 latency however. I think that is the appropriate term to base the pass fail on. What we really want to do is isolate the p99 of the proxy from the p99 of the brokers, which is considerably easier to say than do.